src/components/Articles/Articles.jsx

```diff
@@ -1,11 +1,14 @@
 import "./Articles.css";
-import { getArticles } from "../../utils/api";
+import { getArticleById, getArticles } from "../../utils/api";
 import { useState, useEffect } from "react";
+import { useNavigate } from "react-router-dom";
 import LoadingSpinner from "../LoadingSpinner/LoadingSpinner";
 
-const Articles = () => {
+const Articles = ({ setArticle }) => {
   const [articles, setArticles] = useState([]);
   const [isLoading, setIsLoading] = useState(false);
```

Comment:
You initialise `isLoading` state as false, but if we were to assume that the first render of a component is always first loading information a more appropriate value would be true. 
This would allow you to refactor your useEffect and remove the `setIsLoading(true);` - as this useEffect currently has an empty dependancy array and will only run once there is no need to switch back and forth an extra time.